independent-projects/arc/processor/src/test/java/io/quarkus/arc/processor/AnnotationLiteralProcessorTest.java

```diff
@@ -398,7 +398,7 @@ public void missingAnnotationClass() {
         try (ClassCreator creator = ClassCreator.builder().classOutput(cl).className(generatedClass).build()) {
             MethodCreator method = creator.getMethodCreator("hello", void.class);
 
-            assertThrows(NullPointerException.class, () -> {
+            assertThrows(IllegalArgumentException.class, () -> {
```

Comment:
We can keep the NPE if we want to but I usually stay away from them.

Reply:
I agree that NPEs are evil :)
IAE or ISE are both fine with me.